pkg/commands/models/commit.go

```diff
@@ -29,6 +32,13 @@ func (c *Commit) RefName() string {
 	return c.Sha
 }
 
+func (c *Commit) ParentRefName() string {
+	if len(c.Parents) == 1 && c.Parents[0] == "" {
```

Comment:
could we update `pkg/commands/loaders/commits.go` so that if we're dealing with the initial commit we have the parents field set to an empty slice? The fact it's currently a slice of a single empty string is just due to how we've parsed the commit log output there.

It would also be good to extract this line into its won method i.e.:
```go
func (c *Commit) IsFirstCommit() bool {
  return len(c.Parents) == 0
}
```
Just to make things more explicit